packages/samples/mvp-dapp/src/views/load.view.ts

```diff
@@ -0,0 +1,60 @@
+// import type { StoreType } from '../record/record.model'
+
+type ItemData = {
+  key: string
+  value: string
+  label: string
+}
+
+export type StoreType = {
+  data: {
+    profile: ItemData[]
+    addresses: ItemData[]
+    custom: ItemData[]
+    dweb: ItemData[]
+  }
+}
+
+export class Load {
+  static toJson(storage: StoreType): string {
+    const profile = new Map<string, Item>()
+    storage.data.profile.forEach((item) => {
+      profile.set(item.key, {
+        value: item.value,
+        optional: item.label,
+      })
+    })
+    const addresses = new Map<string, Item>()
+    storage.data.addresses.forEach((item) => {
+      addresses.set(item.key, {
+        value: item.value,
+        optional: item.label,
+      })
+    })
+    const custom = new Map<string, Item>()
+    storage.data.custom.forEach((item) => {
+      custom.set(item.key, {
+        value: item.value,
+        optional: item.label,
+      })
+    })
+    const dweb = new Map<string, Item>()
+    storage.data.dweb.forEach((item) => {
+      dweb.set(item.key, {
+        value: item.value,
+        optional: item.label,
+      })
+    })
+    return JSON.stringify({
+      profile,
+      addresses,
+      custom,
+      dweb,
+    })
+  }
+}
+
+interface Item {
```

Comment:
Declaration `interface Item` could be placed with `type ItemData` and `type StoreType`

Reply:
Just refer to this https://github.com/Magickbase/kuai-mvp-dapp-ui/blob/develop/utils/mock.ts

Reply:
The point is that `type declaration` could be placed together, and should be placed before where it's used.